src/lib.rs

```diff
@@ -317,7 +323,7 @@ fn test_resize_policy() {
 /// }
 /// ```
 #[derive(Clone)]
-pub struct HashMap<K, V, S = RandomState> {
+pub struct HashMap<K, V, S = AdaptiveState> {
```

Comment:
I think the purpose of this repo is to iterate outside std, but with the ultimate goal of incorporating any worthwhile changes back into std.